gems/pending/appliance_console/database_maintenance.rb

```diff
@@ -0,0 +1,61 @@
+require 'appliance_console/logging'
+require 'appliance_console/prompts'
+require 'fileutils'
+
+module ApplianceConsole
+  class DatabaseMaintenance
+    include ApplianceConsole::Logging
+
+    HOURLY_CRON = "/etc/cron.hourly/miq-pg-maintenance-hourly.cron".freeze
+
+    def ask_questions
+      clear_screen
+      confirm
+    end
+
+    def activate
+      say("Configuring Database Maintenance...")
+      if hourly_configured?
+        deactivate_hourly
+      else
+        configure_hourly
+      end
+    end
+
+    private
+
+    def configure_hourly
+      say("Configuring Hourly Database Maintenance...")
+      write_hourly_cron
+      FileUtils.chmod(0755, HOURLY_CRON)
+      true
+    end
+
+    def confirm
+      if hourly_configured?
+        agree("\Hourly Database Maintenance is already configured, Un-Configure (Y/N):")
```

Comment:
@carbonin I put that '\' in there just as a test to see if you would find it. Well done!